ci/build.yml

```diff
@@ -31,3 +32,23 @@ build and push test image:
     - docker build --tag $CONTAINER_NAME:$CI_COMMIT_SHA --tag $CONTAINER_NAME:test -f docker/Dockerfile .
     - docker push $CONTAINER_NAME:$CI_COMMIT_SHA
     - docker push $CONTAINER_NAME:test
+
+# regtest docker images build
+.build_regtest: &build_regtest
```

Comment:
I did it this way in case we decide to do testing image the same way as for the base image. But you're eighth it's not necessary and blockbook is released once a year so that's fine to not have testing image.

Reply:
Aaah I see :+1:. Well I think it's fine then, I was just wondering. Up to you then!